server.go

```diff
@@ -305,21 +305,23 @@ func (s *Server) serveConnection(ctx context.Context, c *serverConn, config serv
 }
 
 func (s *Server) serveRequest(res *responseWriter, req *Request) (err error) {
-	args := req.Args
+	cmds := req.Cmds
```

Comment:
It seems like this intermediary variable isn't useful, we could just write `for _, cmd := range req.Cmds {`